cpu/stm32/include/periph/cpu_i2c.h

```diff
@@ -24,12 +24,30 @@
 #include <stdint.h>
 
 #include "cpu.h"
+#include "macros/units.h"
 #include "periph/cpu_gpio.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#ifdef DOXYGEN
+/**
+ * @brief   Which I2C peripheral variant is used
+ *
+ * If defined as `1` the variant implemented in `i2c_1.c` is used, if
+ * defined as `2` the variant implemented in `i2c_2.c` is used.
+ */
+#  define STM32_I2C_VARIANT 0
+#else
+#  ifdef I2C_CR1_ANFOFF
+/* i2c_1.c is used */
+#    define STM32_I2C_VARIANT 1
+#  else
+/* i2c_2.c is used */
+#    define STM32_I2C_VARIANT 2
+#endif
+
```

Comment:
```suggestion
#endif
```

Reply:
I reverted this, it results in an `#endif` without matching `#if`.